plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -170,7 +170,7 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 
 		const setupList = getTaskList( activeSetupList );
 
-		const isSetupTaskListHidden = setupList?.isHidden;
+		const isSetupTaskListHidden = setupList ? setupList.isHidden : true;
```

Comment:
Nitpicky, but I think this is safer:

```suggestion
		const isSetupTaskListHidden = setupList?.isHidden ?? true;
```

Since otherwise if `setupList = {}`, `setupList ? setupList.isHidden : true` will return `null`

Reply:
Good point! 👍